src/index.html

```diff
@@ -13,6 +13,6 @@
       For production, you should be able to use the same index.html file, but
       this JS resource would be served from a static file.
     -->
-    <script src="/bundle.js"></script>
+    <script src="/bundle.min.js"></script>
```

Comment:
One potential issue with this is that when the `ts_devserver` is serving this file, the bundle is not minified. Calling the script "bundle.min.js" could give readers the wrong idea about what goes on in the `ts_devserver` case.

Reply:
@alexeagle and I discussed this and it's not ideal but I didn't want to use a different html file for the lite-server. Could run the html file through a template but that's another level of complication.